### PR TITLE
Staging deployment of Sprint

### PR DESCRIPTION
**Pull request by vogelino**

### CODE REVIEW

**Comment by Esshahn** [comment]
lol

**Comment by vogelino** [comment]
@ff6347 @Esshahn @julizet @lucasoeth Here are some observations I made while testing the map. Some bugs or UI/UX weaknesses might have already been there. Maybe you can help me prioritize them. In general, I must say that the app feels much better on Desktop. On mobile, well it's all right but I think we can make it feel more made for mobile 😄 

**Bugs:**
- [ ] **NEW** - Filtering for "already adopted" or "recently watered" trees using the search menu does not work on mobile
- [ ] **NEW** - The "Impressum & Datenschutz" link on the bottom right corner is displayed over the overlay and the sidebar
- [ ] **NEW** - Zooming on mobile by double-tapping isn't animated
- [ ] **NEW** - Geolocation on mobile doesn't show a message when geolocalization is not allowed. The spinning target icon keeps spinning indefinitely and there is no way to know why.
- [ ] **NEW** - The cookie banner on mobile looks broken
- [ ] **NEW** - The links in the overlay paragraph are unstyled
- [ ] **OLD** - Clicking on a pump on mobile doesn't show the tooltip

**Possible improvements**
- [ ] Filtering for "already adopted" or "recently watered" shouldn't be possible when the data displayed is the pumps or the rain
- [ ] When focusing on the search field on mobile, the container should scroll to make room for the search results. Otherwise, there is no way for the user to know that the results are further below
- [ ] Once a search result is selected on mobile, the sidebar should close to show the map
- [ ] Make the over/sidebar close button and title sticky
- [ ] The legend on mobile should be closed by default
- [ ] When a tree is selected on mobile, the sidebar should leave room above to show the map so both the map and the tree information can be displayed

**Other remarks:**
- [ ] Should the beta mention still be there?

File: .env.sample [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+# to delete users
+# see https://github.com/technologiestiftung/tsb-trees-api-user-management
+USER_DATA_API_URL=https://my-user-management-api.io
 
-# Trees API
-API_ENDPOINT="https://localhost:8000/my-local-postgres-api" # DEV
-# API_ENDPOINT="https://giessdenkiez-de-postgres-api.vercel.app" # PROD
+# giessdenkiez API
+# See https://github.com/technologiestiftung/giessdenkiez-de-postgres-api
+API_ENDPOINT_DEV=https://localhost:8000/my-local-postgres-api
```
Comment (vogelino) [inline .env.sample:19]: This should be `API_ENDPOINT` right? We don't use both `DEV` and `PROD` anymode.

**Review by dnsos — APPROVED** [review]

**Pull request merged**